.github/workflows/on_build_do_test.yml

```diff
@@ -318,9 +321,14 @@ jobs:
 
       - name: Copy jacoco directories to Infinispan
         working-directory: test_dir/infinispan
+        shell: bash
+        # language=bash
```

Comment:
Comment can be removed.

Reply:
@ryanemerson this comment is for better readability (in Intellij IDEA, when you have this comment it highlights the bash code). We have the same in KCB GHA. But if this is against infinispan coding policy I can remove this.

Reply:
I didn't realise you could do that, it's a nice addition :+1: